frontend/src/components/navigation/UserMeta.vue

```diff
@@ -15,17 +16,19 @@
       </v-toolbar-items>
     </template>
     <v-list dense class="user-nav"
+            tag="ul"
             light color="blue-grey lighten-5">
-      <v-list-item block :to="{ name: 'profile' }">
+      <v-list-item tag="li" block :to="{ name: 'profile' }">
         <v-icon left>mdi-account</v-icon>
-        <span>Profil</span>
+        <span>{{$t('profile')}}</span>
       </v-list-item>
-      <v-list-item block
+      <v-list-item block tag="li"
                    exact :to="{ name: 'camps', params: { groupName: encodeURI('Pfadi Bewegung Schweiz') } }">
         <v-icon left>mdi-format-list-bulleted-triangle</v-icon>
-        <span>Meine Camps</span>
+        <span>{{$tc('myCamps', api.get().camps().items.length)}}</span>
```

Comment:
Cool ein Beispiel zu haben für Plural 👍 
Ich denke in den meisten solchen Fällen, wo der Textbaustein teil eines Titels/Link/Überschrift und nicht Teil eines Satzes ist dürften wir auch pragmatisch sein und einfach den Plural anzeigen.

Reply:
Ja das ist klar. Ist nur ein Beispiel 👍